apps/knowledge-lifeform/obsidian-sync-status.sh

```
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# obsidian-sync-status.sh — 查 vault git 同步狀態 (手機/電腦/Hermex 共用)
+# 輸出 JSON: {vault_clean, unpushed, last_commit, branch}
+set -e
+VAULT="C:/Project/esggo/vault"
+cd "$VAULT" 2>/dev/null || { echo '{"error":"vault not found"}'; exit 1; }
+UNCOMMITTED=$(git status --porcelain | wc -l)
+UNPUSHED=$(git log origin/$(git branch --show-current)..HEAD --oneline 2>/dev/null | wc -l || echo 0)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)